sympy/concrete/summations.py

```diff
@@ -946,9 +946,15 @@ def eval_sum(f, limits):
         return eval_sum_direct(f, (i, a, b))
     if isinstance(f, Piecewise):
         return None
+    #If the number of terms is infinite and series is convergent
+    #if not(f.is_convergent) and expr.subs(1,a+i).is_positive:
+    #    return oo
+    # elif not(f.is_convergent) and expr.subs(i,a+i).is_negative:
+    #    return -oo
```

Comment:
Are these lines left over from debugging?

Reply:
Yes,but this is from other PR.Then,I didn't know how to create branches.So,here is the problem.I will clear this thing over here